### PR TITLE
Feat/netapp scaffolding

### PR DESCRIPTION
**Pull request by JeremyDec**
| Question         | Answer
| ---------------- | ---
| Branch?          | `feat/netapp`
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | #MANAGER-7302
| License          | BSD 3-Clause

## Description

Scaffold NetApp application and module

### CODE REVIEW

File: packages/manager/apps/netapp/package.json [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+    "@ovh-ux/ng-ovh-api-wrappers": "^4.0.4",
+    "@ovh-ux/ng-ovh-http": "^5.0.0",
+    "@ovh-ux/ng-ovh-request-tagger": "^1.1.0",
+    "@ovh-ux/ng-ovh-sso-auth": "^4.2.3",
+    "@ovh-ux/ng-ovh-swimming-poll": "^5.0.0",
+    "@ovh-ux/ng-translate-async-loader": "^2.1.0",
+    "@ovh-ux/ng-ui-router-breadcrumb": "^1.0.0",
+    "@ovh-ux/ng-ui-router-line-progress": "^1.2.2",
+    "@ovh-ux/ufrontend": "^1.1.1 || ^2.0.0",
+    "@ovh-ux/ui-kit": "^4.4.3",
```
Comment (antleblanc) [inline packages/manager/apps/netapp/package.json:37]:
```suggestion
    "@ovh-ux/ui-kit": "^4.6.0",
```

File: packages/manager/apps/netapp/package.json [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+    "angular-resource": "^1.7.8",
+    "angular-sanitize": "^1.7.8",
+    "angular-translate": "^2.18.1",
+    "angular-translate-loader-pluggable": "^1.3.1",
+    "core-js": "^3.6.5",
+    "flatpickr": "~4.5.2",
+    "jquery": "^2.1.3",
+    "lodash-es": "^4.17.15",
+    "oclazyload": "^1.1.0",
+    "ovh-api-services": "^11.0.0",
```
Comment (antleblanc) [inline packages/manager/apps/netapp/package.json:53]:
```suggestion
    "ovh-api-services": "^11.2.0",
```

File: packages/manager/modules/netapp/src/routing.js [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+        .then((services) =>
+          services.length === 0
+            ? {
+                state: 'netapp.onboarding',
+              }
+            : false,
+        ),
+    resolve: {
+      ...ListLayoutHelper.stateResolves,
+      apiPath: () => '/storage/netapp',
```
Comment (antleblanc) [inline packages/manager/modules/netapp/src/routing.js:21]:
```suggestion
      apiPath: () => '/storage',
```

Comment (JeremyDec) [reply packages/manager/modules/netapp/src/routing.js]: Yes it is true for the schema, but not for the real API endpoint. So I've let it as is, and had overridden the schema resolve.

File: packages/manager/apps/netapp/package.json [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+    "dev:watch": "yarn run dev",
+    "start": "lerna exec --stream --scope='@ovh-ux/manager-netapp-app' --include-dependencies -- npm run build --if-present",
+    "start:dev": "lerna exec --stream --scope='@ovh-ux/manager-netapp-app' --include-dependencies -- npm run dev --if-present",
+    "start:watch": "lerna exec --stream --parallel --scope='@ovh-ux/manager-netapp-app' --include-dependencies -- npm run dev:watch --if-present"
+  },
+  "dependencies": {
+    "@ovh-ux/manager-config": "^5.0.0 || ^6.0.0",
+    "@ovh-ux/manager-core": "^12.0.0 || ^13.0.0",
+    "@ovh-ux/manager-error-page": "^2.0.0",
+    "@ovh-ux/manager-netapp": "^0.0.0 || ^1.0.0",
```
Comment (marie-j) [inline packages/manager/apps/netapp/package.json:25]: i don't see any `BREAKING CHANGE` commit

File: packages/manager/modules/netapp/src/index.scss [hunk 5]
```diff
@@ -0,0 +1,5 @@
+@import 'bootstrap4/scss/_functions.scss';
```
Comment (marie-j) [inline packages/manager/modules/netapp/src/index.scss:1]: Shouldn't it be scoped ? At least while there are cohabitation with BS3

Comment (JeremyDec) [reply packages/manager/modules/netapp/src/index.scss:1]: Yes, it should be scoped

**Review by antleblanc — APPROVED** [review]

File: packages/manager/modules/netapp/src/dashboard/controller.js [hunk 6]
```diff
@@ -0,0 +1,10 @@
+export default class OvhManagerNetAppCtrl {
```
Comment (radireddy) [inline packages/manager/modules/netapp/src/dashboard/controller.js:1]:
```suggestion
export default class OvhManagerNetAppDashboardCtrl {
```

File: packages/manager/modules/netapp/src/index.js [hunk 7]
```diff
@@ -0,0 +1,32 @@
...
+const moduleName = 'ovhManagerNetAppLazyLoading';
+
+angular.module(moduleName, ['ui.router', 'oc.lazyLoad']).config(
+  /* @ngInject */ ($stateProvider) => {
+    $stateProvider.state('netapp', {
+      url: '/netapp',
+      template,
+      redirectTo: 'netapp.index',
+      resolve: {
+        breadcrumb: () => 'netapp',
```
Comment (radireddy) [inline packages/manager/modules/netapp/src/index.js:16]: are you planning to translate this later?

File: packages/manager/modules/netapp/src/template.html [hunk 8]
```diff
@@ -0,0 +1 @@
+<div class="netapp-section w-100 h-100" data-ui-view="netappContainer"></div>
```
Comment (marie-j) [inline packages/manager/modules/netapp/src/template.html:1]: Necessary to name the ui-view ?

File: packages/manager/modules/netapp/src/translations/Messages_fr_FR.json [hunk 9]
```diff
@@ -0,0 +1,3 @@
+{
+  "netapp_title": "NetApp"
```
Comment (marie-j) [inline packages/manager/modules/netapp/src/translations/Messages_fr_FR.json:2]: Not sure translation is needed if we keep "NetApp" for all but nitpicking

File: packages/manager/modules/netapp/src/index.scss [hunk 10]
```diff
@@ -0,0 +1,7 @@
+.netapp-section {
```
Comment (marie-j) [inline packages/manager/modules/netapp/src/index.scss:1]: This could have just been applied to netapp component, no need to create a class :)

**Review by marie-j — APPROVED** [review]

File: packages/manager/modules/netapp/src/translations/Messages_fr_FR.json (hunk 9 above)
Comment (JeremyDec) [reply packages/manager/modules/netapp/src/translations/Messages_fr_FR.json:2]: Yes I was thinking the same, but I choose to stick with what is done for other modules, like sms or iplb

File: packages/manager/modules/netapp/src/template.html (hunk 8 above)
Comment (JeremyDec) [reply packages/manager/modules/netapp/src/template.html:1]: Not necessary, it's just an habit :)

File: packages/manager/modules/netapp/src/index.js (hunk 7 above)
Comment (JeremyDec) [reply packages/manager/modules/netapp/src/index.js]: For the moment, a translation is added. For the PR context, no translation demand will be done. We will wait for other netapp features, to check which content should be requested for translation.

**Review by antleblanc — APPROVED** [review]

**Pull request merged**